server/index.ts

```diff
@@ -35,13 +35,10 @@
  * Sockets - from sockets.js
  *
  *   Used to abstract out network connections. sockets.js handles
- *   the actual server and connection set-up.
```

Comment:
? I assume this was a mistake

Reply:
Yeah, I don't know what that's doing there.